Makefile

```diff
@@ -72,23 +86,23 @@ webconsole/default:
 
 .PHONY: immuclient
 immuclient:
-	$(GO) build -v -ldflags '$(V_LDFLAGS_COMMON)' ./cmd/immuclient
+	$(GO) build $(IMMUDB_BUILD_TAGS) -v -ldflags '$(V_LDFLAGS_COMMON)' ./cmd/immuclient
```

Comment:
are build tags relevant for immuclient now?

Reply:
The problem is following:
Statik can only handle multiple files in this way:
https://github.com/rakyll/statik/issues/56

So basically we can do it only at init time

Because of that I had to add imports like
```_ "github.com/codenotary/immudb/webconsole/webconsoleembedded"```

when immudb is builded with let say:
SWAGGER=true WEBCONSOLE=default
It will generate webconsoleembedded content, but only for:
```
//go:build webconsole
// +build webconsole
```
And later on when building immuclient without these tags - it will not include file

that leads into a next problem
```
package github.com/codenotary/immudb/cmd/immuclient
        imports github.com/codenotary/immudb/cmd/immuclient/command
        imports github.com/codenotary/immudb/cmd/immuclient/audit
        imports github.com/codenotary/immudb/pkg/server
        imports github.com/codenotary/immudb/webconsole
        imports github.com/codenotary/immudb/webconsole/webconsoleembedded: build constraints exclude all Go files in /home/razikus/immudb/webconsole/webconsoleembedded
make: *** [Makefile:89: immuclient] Error 1
```

And thats true because its built with different tags, and statik is not generated by default

it will allow to build immuclient only if before immudb was built without tags

Some solutions:
- add tags to makefile (done)
- generate webconsole and swagger defaults always when building other things (basically adding env go xxx to all targets)
- change go 1.15 to go 1.16 to allow go embed https://pkg.go.dev/embed (this will remove this unnecessary import as this support multiple packages out of the box)
- bundle webconsole and swagger together (would rather avoid)

Reply:
actually that's revealing a dependency on webconsole that may not be there...